packages/fleather/lib/src/widgets/editor.dart

```diff
@@ -602,6 +609,11 @@ class RawEditor extends StatefulWidget {
   /// Defaults to false. Must not be null.
   final bool readOnly;
 
+  /// Whether to enable autocorrection.
+  ///
+  /// Defaults to `true`.
+  final bool autocorrect;
```

Comment:
The `FleatherEditor` needs to pass the `autocorrect` to `RawEditor`.

Reply:
Oops! Fixed now.